realm/realm-library/src/main/java/io/realm/RealmObject.java

```diff
@@ -290,7 +291,7 @@ public final void removeChangeListeners() {
             Observable<E> observable = (Observable<E>) realm.configuration.getRxFactory().from(dynamicRealm, dynamicObject);
             return observable;
         } else {
-            throw new UnsupportedOperationException(realm.getClass() + " not supported");
+            throw new UnsupportedOperationException(realm.getClass() + " does not support RxJava.");
```

Comment:
The exception in the RealmObservableFactory throws an message with a pointer to our docs. IMO we should do the same in these now as people new to Realm might not have read the docs and accidentally use this method?